src/widgets/gauge.rs

```diff
@@ -375,4 +363,22 @@ mod tests {
                 .remove_modifier(Modifier::DIM)
         )
     }
+
+    #[test]
+    fn line_gauge_default_is_identical_to_old_default() {
```

Comment:
The naming of this implies that there is something interesting about the old defaults, but it's not obvious what that might be.

Try to phrase tests as defining current behavior unless there's something specifically novel about what the old behavior was (and if that's so, add docs on why that is interesting). Here I think the defaults are just ordinary defaults and `fn default()` would be fine with `assert_eq!(LineGauge::default(), LineGauged { ... }`

Reply:
I got your point, but still have no idea what test name is better, what do you like?

Reply:
I'd go with just `fn default()` as the test name.